src/boot/boot.janet

```diff
@@ -2746,6 +2746,8 @@
                 (and as (string as "/"))
                 prefix
                 (string (last (string/split "/" path)) "/")))
+  (unless (zero? (length prefix))
+    (put-in env [:modules prefix] newenv))
```

Comment:
Why is this here? Is this a local change that got added by accident? This seems like a local hack you added for your repl use-case.

Reply:
I'm sorry; I neglected to explain this in the PR write-up. I thought it would be useful generally to have the environments for modules stored in a dynamic binding in each environment. However, that's a change that's orthogonal to the goal of this PR and so I've removed.